src/components/TextInput/TextInput.module.scss

```diff
@@ -0,0 +1,9 @@
+@import "../../styles/mixins.scss";
+
+.textInput {
+  width: 20rem;
```

Comment:
Why did you hardcoded the width of text input?
Will be great set width: 100%
You can manage the width from parent component